rkt/status.go

```diff
@@ -16,22 +16,156 @@
 
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/appc/spec/schema/types"
+	"github.com/coreos/rocket/pkg/lock"
+)
 
 var (
 	cmdStatus = &Command{
 		Name:    "status",
 		Summary: "Check the status of a rkt job",
-		Usage:   "UUID",
+		Usage:   "--uuid UUID [--wait]",
 		Run:     runStatus,
 	}
+	flagContainerUUID types.UUID
+	flagWait          bool
+)
+
+const (
+	statusDir = "stage1/rkt/status"
 )
 
 func init() {
 	commands = append(commands, cmdStatus)
+	cmdStatus.Flags.Var(&flagContainerUUID, "uuid", "uuid of container to query the status of")
```

Comment:
I had originally implemented it that way, but preferred the agnostic ordering UX which was non-trivial without both being flags with the current way flags are being parsed.

I'll change it back and just enforce ordering of --wait coming before UUID.